generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.reducer.ts.ejs

```diff
@@ -177,6 +205,16 @@ export const deleteUser: ICrudDeleteAction<IUser> = id => async dispatch => {
   return result;
 };
 
+export const getResetCode = mail => ({
+  type: ACTION_TYPES.RESET_CODE,
+  payload: axios.post('/api/account/reset-password/code', mail, { headers: { 'Content-Type': 'text/plain' } })
+});
+
+export const getMailStatus = () => ({
```

Comment:
You should reuse the profileInfo reducer for this and avoid this action, as it already calls the API and has the info